README.md

```diff
@@ -1,7 +1,91 @@
-# Fabric
+# Fabric API
 
-Essential hooks and patches for modding with Fabric.
+Essential hooks for modding with Fabric.
 
-## Getting Started
+Fabric API is the library for essential hooks and interoperability mechanisms for Fabric mods. Examples include:
 
-To setup a Fabric development environment, check out the [example fabric mod](https://github.com/FabricMC/fabric-example-mod) and follow the instructions there.
+- Exposing functionality that is useful but difficult to access for many mods such as particles, biomes and dimensions
+- Adding events, hooks and APIs to improve interopability between mods.
+- Essential features such as registry synchronization and adding information to crash reports.
+- An advanced rendering API designed for compatibility with optimization mods and graphics overhaul mods.
+
+Also check out [Fabric Loader](https://github.com/FabricMC/fabric-loader), the (mostly) version-independent mod loader that powers Fabric. Fabric API is a mod like any other Fabric mod which requires Fabric Loader to be installed.
+
+For support and discussion for both developers and users, visit [the Fabric Discord server](https://discord.gg/v6v4pMv).
+
+## Using Fabric API
+
+To use Fabric API, download it from [CurseForge](https://www.curseforge.com/minecraft/mc-mods/fabric-api) or [GitHub Releases](https://github.com/FabricMC/fabric/releases).
+
+The downloaded jar file should be placed in your `mods` folder.
+
+## Getting Started With Development
+
+To setup a Fabric development environment, check out the [Fabric example mod](https://github.com/FabricMC/fabric-example-mod) and follow the instructions there. The example mod already depends on Fabric API.
+
+To include all of Fabric API in a mod development environment, add the following to your `dependencies` block in the gradle buildscript:
+
+### Groovy DSL
+
+```groovy
+modImplementation "net.fabricmc.fabric-api:fabric-api:FABRIC_API_VERSION"
+```
+
+### Kotlin DSL
+
+```kotlin
+modImplementation("net.fabricmc.fabric-api:fabric-api:FABRIC_API_VERSION")
+```
+
+Individual modules from Fabric API may be resolved via the `fabricApi` extension in Loom for a smaller dependency footprint:
```

Comment:
This isn't very clear IMO, one may not recognize that "all of Fabric API" above means all modules and that the individual modules added here are related.

I'd change the wording to something like "To include the full Fabric API with all modules in a mod [...]" and "Alternatively, modules from Fabric API can be specified individually via the [...]"